modules/apps/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/display/context/SelectSiteInitializerDisplayContext.java

```diff
@@ -150,7 +150,7 @@ private List<SiteInitializerItem> _getSiteInitializerItems()
 						themeDisplay.getCompanyId(), Boolean.TRUE, null),
 					layoutSetPrototype -> new SiteInitializerItem(
 						layoutSetPrototype, themeDisplay.getLocale())),
-				new SiteInitializerNameComparator(true));
+				SiteInitializerNameComparator.getInstance(true));
```

Comment:
@victorhcunha This class is clearly only used as a collection comparator, so it doesn't need to be an `OrderByComparator` with singleton pattern. Please make it an ordinary `java.util.Comparator`, keep direct constructor usage.